cmd/update-notifier.go

```diff
@@ -27,6 +27,61 @@ import (
 	"github.com/fatih/color"
 )
 
+// humanizeDuration converts time.Duration to human readable duration.
+func humanizeDuration(duration time.Duration) (durationStr string) {
```

Comment:
Humanization is slight different here.  As this affects UI, it would also be reviewed by @abperiasamy